### DISCUSSION TITLE
Argo-cd deployment rollout  from UI

### DISCUSSION THREAD

**Discussion by byvinx** [argoproj/argo-cd#11046] [General]
Hi, I used the command: **argocd app actions run app-name restart --kind Deployment --all**  to roll out the deployment, which is necessary in many use cases. There is the possibility to do the rollout from the argo-cd web UI? Thank you.

**Comment by alexmt** [comment]
The `Restart` command should be available in Argo CD UI as well. Try to click `...` button of required Deployment on application details page:

<img width="444" alt="image" src="https://user-images.githubusercontent.com/426437/197553520-4ad73943-670f-490f-b413-16f705772b79.png">

**Comment by byvinx** [comment]
Great. I didn't see it, in the visualization I have to reduce the browser to less than 100%. otherwise it is hidden. Res: 1920x1080
![Cattura](https://user-images.githubusercontent.com/7317029/197712218-153f6fb1-1f8a-49e7-9229-5932c770ae28.JPG)
